test-cases/tests/monitoring/m05-verify-secrets.md

```diff
@@ -0,0 +1,47 @@
+---
+targets:
+  - 2.6.0
+---
+
+# M05 - Verify secrets
+
+More info:
+
+- <https://issues.redhat.com/browse/INTLY-4885>
+- <https://issues.redhat.com/browse/INTLY-9096>
+
+## Steps
+
+1. Login into OpenShift console as kubeadmin
```

Comment:
@Rajagopalan-Ranganathan would you mind reviewing these test case steps for accuracy

Reply:
Looks good to me.